packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -397,6 +402,45 @@ class WebView extends StatefulWidget {
   /// By default `userAgent` is null.
   final String? userAgent;
 
+  /// (Android only) Sets whether the WebView should support zooming using its on-screen zoom controls and gestures.
+  ///
+  /// The particular zoom mechanisms that should be used can be set with setBuiltInZoomControls(boolean).
+  ///
+  /// By default 'setSupportZoom` is true
+  final bool setSupportZoom;
+
+  /// (Android only) Sets whether the WebView should use its built-in zoom mechanisms.
+  ///
+  /// The built-in zoom mechanisms comprise on-screen zoom controls, which are displayed over the WebView's content, and the use of a pinch gesture to control zooming.
+  ///
+  /// By default 'setBuiltInZoomControls` is true
+  final bool setBuiltInZoomControls;
```

Comment:
Short answer- no. If either of those are set to false, then zooming just doesn't work

As you pointed out in https://github.com/flutter/plugins/pull/3325#discussion_r566442302, it does feel like not all of these Android-specific params need to be exposed. 

I think it would definitely simplify things, and and allow for a more consistent API, if only the following were shown:

`this.zoomEnabled = true` 
`this.loadWithOverviewMode = true` This could even potentially be set to default (true) in native

Then these would be set to default values inside the native Android code
`this.builtInZoomControlsEnabled = true`  confusingly redundant anyway
`this.displayZoomControls = false` deprecated anyway
`this.useWideViewPort = true` just seems like a good default on mobile anyway

**Before I proceed with making that change, I want to make sure you agree with one of these choices below** 

A) Exposing the two params outlined above OR
B) Only exposing `zoomEnabled` and just setting default values for the others in native Android code